src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 50px;
+  background-color: #f0f0f0;
```

Comment:
The color `#f0f0f0` should be placed in the variables file for maintainability and reusability according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).